### PR TITLE
Eric Mielke's IC

### PR DESCRIPTION
**Pull request by EMielke76**

### CODE REVIEW

File: lib/venue.rb [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+      patron.upcase
+    end
+  end
+
+  def over_capacity?
+    @patrons.length > @capacity
+  end
+
+  def kick_out
+    @patrons.pop
```
Comment (dcoleman21) [inline lib/venue.rb:27]: This works but how can we check when we would need it to remove a patron or not? Could you use a `loop` here? So something like 

```
until over_capacity? == false
   @patrons.pop
end

while over_capacity? == true
    @patrons.pop
end
```

File: spec/venue_spec.rb [hunk 2]
```diff
@@ -44,12 +44,45 @@
...
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  # Iteration III
+
+  describe '#over_capacity?' do
```
Comment (dcoleman21) [inline spec/venue_spec.rb:58]: Very nice tests.

File: lib/venue.rb [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+    @capacity = capacity
+    @patrons  = []
+  end
+
+  def add_patron(name)
+    @patrons << name
+  end
+
+  def yell_at_patrons
+    @patrons.map do |patron|
```
Comment (dcoleman21) [inline lib/venue.rb:17]: Great use of `.map`

File: lib/venue.rb [hunk 4]
```diff
@@ -1,2 +1,29 @@
 class Venue
+  attr_reader :name,
```
Comment (dcoleman21) [inline lib/venue.rb:2]: Great job 😃

**Review by dcoleman21 — COMMENTED** [review]
Really great job on this first challenge. It shows that you really continued to dive into this stuff while you were away. I left you some feedback to review.

File: lib/venue.rb (hunk 1 above)
Comment (EMielke76) [reply lib/venue.rb:27]: Yo, good call. It didn't even occur to me that I should have made that conditional. Thank you!

**Pull request closed without merging**